frontend/src/pages/exploreApplication/EnableModal.tsx

```diff
@@ -18,6 +19,7 @@ const EnableModal: React.FC<EnableModalProps> = ({ selectedApp, shown, onClose }
   const [postError, setPostError] = React.useState('');
   const [validationInProgress, setValidationInProgress] = React.useState(false);
   const [enableValues, setEnableValues] = React.useState<{ [key: string]: string }>({});
+  const [isEnableValuesHasEmptyValue, setIsEnableValuesHasEmptyValue] = React.useState(true);
```

Comment:
One last suggestion, I think you can change the from a `React.useState` into a `React.useMemo` 
```suggestion
  const isEnableValuesHasEmptyValue = React.useMemo(() => values(updatedValues).some((val) => isEmpty(val)), [updatedValues]);
```
 
This would reduce the need to keeping track of another state and potential mistakes with them getting out of sync. Then you don't need to manually set it to `true` or `false` as it will be done automatically

Reply:
Great suggestion! Thanks!